docs/parallel/amp/cpp-amp-overview.md

```diff
@@ -7,6 +7,10 @@ ms.assetid: 9e593b06-6e3c-43e9-8bae-6d89efdd39fc
 ---
 # C++ AMP Overview
 
+> [!NOTE]
+> C++ AMP headers are deprecated, starting with Visual Studio 2022 version 17.0.
+> Including any AMP headers will generate build errors. To Silence the errors, define "_SILENCE_AMP_DEPRECATION_WARNINGS" before including the headers.
```

Comment:
Small typo (Silence). If you want, you could be more direct, and also use code escape around the define (the backticks)

Including AMP headers will now generate build errors. Define `_SILENCE_AMP_DEPRECATION_WARNINGS` before including any AMP headers to silence warnings.